test/test_entity-manager.js

```diff
@@ -182,6 +182,36 @@ define(function (require) {
                 expect(manager.processors[0].entities).to.have.length(1);
                 expect(manager.processors[0].entities[0]).to.equal(newEntityId);
             });
+
+            it('does not sends the update event from manager to processors', function () {
+                var manager = prepareManager();
+                var newEntityId = 42;
+
+                var processor = {
+                    entities: [],
+                    on: function (type, data) {
+                        switch (type) {
+                            case 'COMPONENT_UPDATED':
+                                this.entities.push(data.entity);
+                                break;
+                            default:
+                                break;
+                        }
+                    },
+                    update: function () {},
+                };
+                manager.addProcessor(processor);
+
+                var entity = manager.createEntity(['Position'], newEntityId);
```

Comment:
I think this should also test the case where you send initial data in `createEntity`, as that's really why you made the code in the first place.

Reply:
Yeah right. Should be ok now.